### PR TITLE
Change npx serverless To npx components-v1

### PR DESCRIPTION
**Pull request by ivyastrix**
### What changes were made?

- Changes to enable usage of components command line rather than serverless command line. 

### Is there any background info about the change?

- Due to versioning issues with the nextjs serverless project we have to use components-v1 instead of serverless at hte commadn line. 

### Are there any state changes? Mention key ones (if any)

- Precursor to having different serverless.yml files for each branch (excluded from merges via git attributes)

### Shortcut Link

https://app.shortcut.com/twali/

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -1,18 +1,12 @@
-twali-xyz-production:
+twali-xyz-staging:
   component: "@sls-next/serverless-component@3.7.0-alpha.11"
   inputs:
     roleArn: "arn:aws:iam::517934542124:role/deploy-dev"
     build:
       env:
-        TABLE_NAME: "Twali-core-test"
+        TABLE_NAME: "staging_twali_core"
         WHITELIST_TABLE: "profiles_whitelist_staging_test"
```
Comment (NickGonzalez04) [inline serverless.yml:8]: Just need to remove `_test` at the end.

**Review by NickGonzalez04 — CHANGES_REQUESTED** [review]
Looks good on my end

File: serverless.yml (hunk 1 above)
Comment (ivyastrix) [reply serverless.yml]: done!

**Review by NickGonzalez04 — APPROVED** [review]
Good to gooo 🚀

**Pull request merged**